controllers/handler/node_controller.go

```diff
@@ -117,10 +118,11 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, request ctrl.Request) (c
 }
 
 func (r *NodeReconciler) getDependencyVersions() *nmstate.DependencyVersions {
-	handlerNetworkManagerVersion, err := nmstate.ExecuteCommand("NetworkManager", "--version")
+	handlerNetworkManagerVersion, err := nmstate.ExecuteCommand("nmcli", "--version")
 	if err != nil {
 		r.Log.Info("error retrieving handler NetworkManager version: %s", err.Error())
 	}
+	handlerNetworkManagerVersion = strings.TrimPrefix(handlerNetworkManagerVersion, "nmcli tool, version ")
```

Comment:
It may be better to split the output by a space and pick the last element, so we don't depend on the string to stay the same:
 
```go
s := strings.Split(handlerNetworkManagerVersion, " ")
// also check s contains at least 1 element after splitting
handlerNetworkManagerVersion = handlerNetworkManagerVersion[len(s)-1]

```